pkgs/shells/zsh/agdsn-zsh-config/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, zsh
+, coreutils
+, inetutils
+, procps
+}:
+
+with lib;
+
+stdenv.mkDerivation rec {
+  pname = "agdsn-zsh-config";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "agdsn";
+    repo = "agdsn-zsh-config";
+    rev = "v${version}";
+    sha256 = "sha256-/l2fE4ZsZ6f89fYG9sTEV1mrXZ3MLXx6K3CTUQHiAsc=";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    install -D -m644 zshrc-base-hw.zsh $out/etc/zsh/zshrc
+    install -D -m644 zshrc-home.zsh $out/etc/skel/.zshrc
+    install -D -m644 zshrc-home.zsh $out/etc/zsh/newuser.zshrc.recommended
+    install -D -m644 profile-d-agdsn-zsh-config.sh $out/etc/profile.d/agdsn-zsh-config.sh
+  '';
+
+  meta = with lib; {
+    description = "A modified version of the Grml Zsh configuration specialised for the needs of system administration";
+    homepage = "https://github.com/agdsn/agdsn-zsh-config";
+    license = licenses.gpl2;
```

Comment:
Use `gpl2Plus` (https://github.com/agdsn/agdsn-zsh-config/blob/master/debian/copyright), `gpl2` is deprecated.